.github/workflows/ci.yml

```
@@ -0,0 +1,46 @@
+      run: pnpm lint
+
+    - name: Type check
+      run: pnpm build
+
+    - name: Check for security vulnerabilities
+      run: |
+        # Additional security checks
+        echo "Checking for hardcoded secrets..."
+        if grep -r "sk-" --include="*.ts" --include="*.js" --include="*.tsx" --include="*.jsx" app/ components/ lib/ || true; then
+          echo "Warning: Potential API keys found in code"
+        fi 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)